### DISCUSSION TITLE
Error while installing through ESPHome

### DISCUSSION THREAD

**Discussion by jgosnell56** [ajfriesen/ESPHome-AirGradient#22] [Q&A]
When trying to install, I am getting an error.  Any ideas how to resolve this?  I believe the conflict with another device already managed by ESPHome.

`INFO ESPHome 2023.9.3
INFO Reading configuration /config/esphome/airgradient.yaml...
Failed config

button.template: [source /config/esphome/airgradient.yaml:27]
  platform: template
  name: Flash mode
  id: flash_button
  icon: mdi:cellphone-arrow-down
  on_press: 
    then: 
      - 
        ID 'safe_mode' conflicts with the name of an esphome integration, please use another ID name.
        switch.toggle: safe_mode`

**Comment by michaelw** [comment]
You can replace it with this: https://github.com/ajfriesen/ESPHome-AirGradient/blob/main/air-gradient-pro-diy.yaml#L70-L73
and the error will go away.
